ubuntu-package-builder/Dockerfile

```diff
@@ -73,8 +102,16 @@ RUN echo "deb http://ftp.debian.org/debian jessie-backports main" > /etc/apt/sou
         automake \
         autotools-dev
 
-COPY build.sh /
 RUN mkdir -p /workspace
 WORKDIR /workspace
 
-ENTRYPOINT ["/build.sh"]
+COPY debian /workspace/debian
```

Comment:
Isn't `/workspace` mounted over during the cloud build process?

Reply:
right, removed in the next cl